src/plugins/semlock/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-find_package(Threads)
-include (LibAddMacros)
+#find_package(Threads)
```

Comment:
Why are you commenting out the plugin? We already removed the test.
You can propose to completely remove the plugin if you think it is broken by design.
And please do not mix completely unrelated things within a PR.

Imho the semlock plugin would still be useful for setups where config files are on NFS and locking does not work otherwise.

Reply:
I tried to fix the `elektra-multiconfig-gcc-stable` job that is broken due to `testscr_check_kdb_internal_check` that checks the semlock plugin even when the `ADD_TEST` is not present in the CMakeList of semlock, the test cases takes all added Plugins so I tried to not add it.
I found a better solution, I will revert these changes and make a own PR for the better solution.

Reply:
see #1798.